dist/sass/calcite-web/utils/_overflow.scss

```diff
@@ -16,9 +16,3 @@
 %overflow-auto {
   @include overflow-auto();
 }
-
-@if $include-utils == true or $include-drawers == true or $include-modal == true {
```

Comment:
I think we'll still need these util classes

Reply:
@paulcpederson, do I just remove `or $include-drawers == true`?

Reply:
yeah, you can just remove the check for the variable